src/proxy-ui-api/frontend/src/views/Clients/Client.vue

```diff
@@ -27,10 +50,19 @@ export default Vue.extend({
   data() {
     return {
       tab: null,
+      confirmUnregisterClient: false,
+      unregisterLoading: false,
     };
   },
   computed: {
     ...mapGetters(['client']),
+    showUnregister(): boolean {
+      if (this.$store.getters.hasPermission(Permissions.SEND_CLIENT_DEL_REQ)) {
```

Comment:
`return this.$store.getters.hasPermission(Permissions.SEND_CLIENT_DEL_REQ;`

Reply:
Fixed